client/src/components/ProductsList.js

```diff
@@ -65,79 +76,113 @@ export default class ProductsList extends Component {
         })
       )
     ];
-    console.log(distinctCategory);
 
-    const filteredProduct = this.state.products.filter(product => {
+    const distinctBrand = [
+      ...new Set(
+        this.state.products.map(product => {
+          return product.brand;
+        })
+      )
+    ];
+
+    const maxPrice = Math.max(
```

Comment:
Is it possible to provide a max price range 0 - 20.000 instead of max divided through products?